cmd/influxd/backup/backup.go

```diff
@@ -166,5 +323,26 @@ backup downloads a snapshot of a data node and saves it to disk.
         -host <host:port>
                           The host to connect to snapshot.
                           Defaults to 127.0.0.1:8088.
+        -database <name>
```

Comment:
If you set help messages for these options when declaring the flags, you can get this output by calling `PrintDefaults`. https://github.com/influxdb/influxdb/blob/master/cmd/influx_tsm/main.go#L60

It can be more maintainable that way.